src/style.css

```diff
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+}
+
```

Comment:
I see you use a lot the display flex inside the containers, if you create a class like; 
.flex {display: flex;}

you will save a couple of lines of code